yolov6/logger/wandb_inference_logger.py

```diff
@@ -0,0 +1,56 @@
+import cv2
+import wandb
+
+
+class WandbInferenceLogger:
+    def __init__(self) -> None:
+        self.label_dictionary = {}
+        self.table = wandb.Table(
+            columns=[
+                "Image-File",
+                "Predictions",
+                "Number-of-Objects",
+                "Prediction-Confidence",
+            ]
+        )
+
+    def set_label_dictionary(self, label_dictionary):
+        self.label_dictionary = label_dictionary
+
+    def in_infer(self, image, image_file, detection_results):
+        bbox_data, confidences = [], []
+        image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+        height, width, _ = image.shape
```

Comment:
I guess if you just pass the image path as reference as i suggested in another comment, then you'll also need the image dims to calculate the bounding box.

Reply:
Yeah, replied about this in the previous comment.